docs/sources/alerting/_index.md

```diff
@@ -2,39 +2,71 @@
 aliases:
   - /docs/grafana/latest/alerting/
   - /docs/grafana/latest/alerting/unified-alerting/alerting/
+  - /docs/grafana/latest/about-alerting
 title: Alerting
 weight: 114
 ---
 
 # Grafana Alerting
 
-Grafana Alerting allows you to learn about problems in your systems moments after they occur. Robust and actionable alerts help you identify and resolve issues quickly, minimizing disruption to your services. It centralizes alerting information in a single, searchable view that allows you to:
+Grafana Alerting allows you to learn about problems in your systems moments after they occur. Create, manage, and take action on your alerts in a single, consolidated view, and improve your team’s ability to identify and resolve issues quickly.
 
-- Create and manage Grafana alerts
-- Create and manage Grafana Mimir and Loki managed alerts
-- View alerting information from Prometheus and Alertmanager compatible data sources
+Depending on your use case, you can use Grafana Alerting for Grafana OSS, Grafana Enterprise, or Grafana Cloud. With Grafana OSS, you can create Grafana managed alerts or if your data already lives in Grafana Mimir or Grafana Loki, you can run alert expressions closer to your data and at massive scale, all managed by the Grafana UI.
```

Comment:
Grafana Alerting is available for for Grafana OSS, Grafana Enterprise, or Grafana Cloud. With Mimir and Loki alert rules you can run alert expressions closer to your data and at massive scale, all managed by the Grafana UI you are already familiar with.
```suggestion
Grafana Alerting is available for for Grafana OSS, Grafana Enterprise, or Grafana Cloud. With Mimir and Loki alert rules you can run alert expressions closer to your data and at massive scale, all managed by the Grafana UI you are already familiar with.
```